packages/happn-cluster/lib/services/replicator.js

```diff
@@ -174,8 +174,7 @@ Replicator.prototype.emitSecurityDataChanged = function (payload, isLocal, origi
 };
 
 Replicator.prototype.__defaults = function (callback) {
-  if (!this.config.securityChangesetReplicateInterval) {
-    this.config.securityChangesetReplicateInterval = 3e3;
-  }
+  if (!this.config.securityChangesetReplicateInterval)
+    this.config.securityChangesetReplicateInterval = 3000;
```

Comment:
change not necessary

Reply:
Rolled back. Not exactly sure when/why this happened,